tests/unit/plugins/inventory/test_flightctl.py

```
@@ -530,6 +535,359 @@ def test_validate_device_partial_expression_with_missing_field(self):
+    def test_auth_config_failure_raises(self, mock_open):
+        mock_open.side_effect = Exception("Connection refused")
+        with self.assertRaises(ValidationException) as ctx:
+            self.module._oidc_password_grant(
+                "https://host", "admin", "pass", False
+            )
+        self.assertIn("Failed to fetch auth config", str(ctx.exception))
+
+    @patch(OPEN_URL)
+    def test_no_oidc_provider_raises(self, mock_open):
+        mock_open.return_value = self._mock_response({"providers": []})
+        with self.assertRaises(ValidationException) as ctx:
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)